### PR TITLE
[FIX] Widgets update

### LINKED ISSUE #11: Widgets need updating
Both widgets need updating due to changes in Orange.
Fix Error and Warning to classes, check QVariant and setting up AnyQt. All warnings found in the console when using the widgets.

### PR DESCRIPTION
**Pull request by jerneju**
Fixes #11.
+`.gitignore`
+github templates

### CODE REVIEW

File: orangecontrib/associate/widgets/owassociate.py [hunk 1]
```diff
@@ -8,31 +8,37 @@
...
     icon = 'icons/AssociationRules.svg'
     priority = 20
 
-    inputs = [("Data", Table, 'set_data')]
-    outputs = [(Output.DATA, Table)]
+    class Inputs:
+        data = Input("Data", Table)
+
+    class Outputs:
+        data = Output("Matching data", Table)
```
Comment (kernc) [inline orangecontrib/associate/widgets/owassociate.py]: Matching Data

Comment (jerneju) [reply orangecontrib/associate/widgets/owassociate.py]: Fixed.

File: orangecontrib/associate/widgets/owassociate.py [hunk 2]
```diff
@@ -8,31 +8,37 @@
...
     icon = 'icons/AssociationRules.svg'
     priority = 20
 
-    inputs = [("Data", Table, 'set_data')]
-    outputs = [(Output.DATA, Table)]
+    class Inputs:
+        data = Input("Data", Table)
+
+    class Outputs:
+        matching_data = Output("Matching data", Table)
```
Comment (kernc) [inline orangecontrib/associate/widgets/owassociate.py]: Sorry, I meant the labels. We title case them.

Comment (jerneju) [reply orangecontrib/associate/widgets/owassociate.py]: Ok.

**Comment by ajdapretnar** [comment]
Thank you for this! I have some extra requests, if possible.
1) Add a newline break in tooltip, e.g. replace in join. (l:206 in owitemsets.py)
2) Solve QVariant warnings in console. (if possible)
3) Add a warning for sparse data. ("Can't induce rules on sparse data (yet).")
4) Is it possible to fix Find Itemsets/Find Rules so that when checking the box the itemsets/rules are found automatically?

**Comment by jerneju** [comment]
@ajdapretnar : Ok, but I will open a new PR.

**Comment by ajdapretnar** [comment]
Number 2 belongs here I think (error solving). The rest can go into a separate PR.

**Comment by kernc** [comment]
Agree bullets above are out of scope.

**Comment by ajdapretnar** [comment]
Except that QVariant wasn't solved, but who cares...

**Comment by kernc** [comment]
Warning is not an error nor necessarily a bug. If the warning in question is:

> Trying to create a QVariant instance of QMetaType::Void type, an invalid QVariant will be constructed 

I'm happy to take it in a new PR as it might not be as trivial as this was.

**Comment by ajdapretnar** [comment]
#11 refers to warnings, including QVariant. If it is not trivial, this should be stated, since it would seem here that the entirety of #11 was solved (which it wasn't).

**Comment by jerneju** [comment]
I agree, it is not trivial.

**Pull request merged**